packages/perseus/src/widgets/graded-group-set.jsx

```diff
@@ -33,35 +33,41 @@ type IndicatorsProps = {|
     onChangeCurrentGroup: (groupNumber: number) => void,
 |};
 
-// TODO(jeremy): This indicator panel is not accessible for keyboard
-// navigation. Add in keyboard handling and tab indexes to make accessible.
 class Indicators extends React.Component<IndicatorsProps> {
-    render(): React.Node {
-        const items = [];
-        for (let i = 0; i < this.props.numGroups; i++) {
```

Comment:
I think this change makes a lot of sense - why use two props when you can use one? Are we done with `numGroups` now? Could we remove it from props entirely?